synapse/storage/schema/full_schemas/54/full.sql.sqlite

```diff
@@ -0,0 +1,260 @@
+CREATE TABLE application_services_state( as_id TEXT PRIMARY KEY, state VARCHAR(5), last_txn INTEGER );
```

Comment:
Would be good to just have a comment at the top to say these are auto-generated files and people should look at the delta files for comments